.gitignore

```diff
@@ -2,3 +2,4 @@
 .Rhistory
 .RData
 .Ruserdata
+.DS_Store
```

Comment:
You may want to ignore it globally: https://yihui.name/en/2017/09/ds-store-crap/

I'm okay with this particular change, but in general, I recommend that you do one thing (and only one) per pull request: https://yihui.name/en/2018/02/bite-sized-pull-requests/